cleverhans/attacks.py

```diff
@@ -198,7 +198,7 @@ def get_or_guess_labels(self, x, kwargs):
         The kwargs are fed directly from the kwargs of the attack.
         If 'y' is in kwargs, then assume it's an untargeted attack and
         use that as the label.
-        If 'y_target' is in kwargs, then assume it's a targeted attack and
+        If 'y_target' is in kwargs and isn't None, then assume it's a targeted attack and
```

Comment:
reformat this line. pep8 check is failing because of this right now.

Reply:
Gotcha. Thanks for letting me know.

Reply:
Can you please close this pull request? I have created another one that fixes the issue.

Reply:
Can you please close this pull request? I have created another one. @giuscri

Reply:
You should learn about _rebasing_. By the way, it's up to you to close this PR.

Reply:
Oh ok